resolver/manual/manual.go

```diff
@@ -26,7 +26,7 @@ import (
 	"google.golang.org/grpc/resolver"
 )
 
-// NewBuilderWithScheme creates a new test resolver builder with the given scheme.
+// NewBuilderWithScheme creates a new manual resolver builder with the given scheme.
```

Comment:
Should we document that this may only be used with a single `ClientConn`?  Otherwise bad things happen.

Reply:
Done.